ui/src/importer/importerTaskDetails/ImportedTable.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Pagination, Table } from 'semantic-ui-react';
+import _isEmpty from 'lodash/isEmpty';
+import { EitemImportDetailsModal } from '../EitemImportDetailsModal';
+import { SeriesImportDetailsModal } from '../SeriesImportDetailsModal';
+import { JsonViewModal } from '../JsonViewModal';
+import { ImportedDocumentReport } from './ImportedDocumentReport';
+
+export class ImportedTable extends React.Component {
+  constructor(props) {
+    super(props);
+    this.pageSize = 100;
+    const { records } = this.props;
+    this.state = {
+      activePage: 1,
+      records: records,
+    };
+  }
+
+  static getDerivedStateFromProps(props, state) {
```

Comment:
As pointed previously, I didn't manage to find a better way to update the activePage (and with that the state) every time the records changed. Getting the activePage out of the state isn't a good solution also, as it allows to update the page with each page change. Regardless if I receive feedback or not I will update this code, as the check is open for bugs if both records are different but have the same length.